Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -156,7 +176,7 @@ public static int MaxMemoryCacheSize
             catch (Exception ex)
             {
                 System.Diagnostics.Debug.WriteLine(ex.Message);
-                return null;
+                throw ex;
```

Comment:
It is strange but I didn't. I thought it is not enforced for EventArgs derived class or something.